build.sbt

```diff
@@ -67,8 +67,9 @@ lazy val commonSettings = Seq(
   scalacOptions in Tut --= Seq("-Ywarn-unused-import", "-Xmacro-settings:materialize-derivations"),
 
   // use sbt <module_name>/test:console to run an ammonite console
-  libraryDependencies += "com.lihaoyi" % "ammonite" % "1.0.3" % "test" cross CrossVersion.patch,
-  initialCommands in (Test, console) := """ammonite.Main().run()""",
+  // not published for Scala 2.12.6 (maybe now there's another way of integrating this?)
```

Comment:
Should we just dial back to using ammonite with 2.11.12 or pull this code out entirely?

Reply:
I'm OK killing it and looking back in the git history if we need to figure out how to restore it.

Reply:
I'm also OK with removing it.

Reply:
Ok, I removed it in 8368801ad94cda35cf0602840f3d05e8f9507201 then! We can always turn it on again later by looking at the Git history, as @leifwickland said.